Lib/fontTools/merge.py

```diff
@@ -780,6 +849,7 @@ def merge(self, fontfiles):
 		# TODO Is it necessary to reload font?  I think it is.  At least
 		# it's safer, in case tables were loaded to provide glyph names.
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		self.fonts = copy.deepcopy(fonts)
```

Comment:
It's me that is drunk.. I see now where those attribute are beign are used; I was confused because grepping for `self.fonts` would not return any result; the instance was called `m`, so `m.fonts` and `m.fontfiles`... Sorry!